crates/wasi-crypto/src/wiggle_interfaces/asymmetric_common.rs

```diff
@@ -263,8 +265,10 @@ impl From<guest_types::PublickeyEncoding> for PublicKeyEncoding {
             guest_types::PublickeyEncoding::Pkcs8 => PublicKeyEncoding::Pkcs8,
             guest_types::PublickeyEncoding::Pem => PublicKeyEncoding::Pem,
             guest_types::PublickeyEncoding::Sec => PublicKeyEncoding::Sec,
-            guest_types::PublickeyEncoding::CompressedSec => PublicKeyEncoding::CompressedSec,
+            //guest_types::PublickeyEncoding::CompressedSec => PublicKeyEncoding::CompressedSec,
             guest_types::PublickeyEncoding::Local => PublicKeyEncoding::Local,
+
+            _ => todo!("From guest_type::PublicKeyEncoding to PublicKeyEncoding"),
```

Comment:
raw, pkcs8, pem, sec and local are the only possible values. Why is this `todo!()` required?

Reply:
Removed. Previously I was getting errors about unhandled patterns in the `match` statement, but that seems to be resolved.